### PR TITLE
added fetchTransactionHistory function

### PR DESCRIPTION
**Pull request by Liubyvyi**

### CODE REVIEW

File: tests/fetch_transaction_history.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        'Access-Control-Allow-Origin',
+        '*',
+        'X-Ratelimit-Remaining',
+        '98' ])
+
+  let getter = new FetchTransactionHistory()
+  return getter.getFromBlockchain('mjw2BcBvNKkgLvQyYhzRERRgWSUVG7HHTb').then((utxos) => {
+    expect(utxos.txrefs.length).toBe(1)
+  })
+})
```
Comment (MaxFrolov) [inline tests/fetch_transaction_history.test.js]: new line required

**Review by MaxFrolov — APPROVED** [review]

File: tests/fetch_transaction_history.test.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import FetchTransactionHistory from '../src/fetch_transaction_history'
+import nock from 'nock'
+
+test('load transaction history by a given address', () => {
+
+  nock('https://api.blockcypher.com:443', {"encodedQueryParams":true})
+    .get('/v1/btc/test3/addrs/mjw2BcBvNKkgLvQyYhzRERRgWSUVG7HHTb/full')
+    .query({"token":process.env.BLOCKCYPHER_TOKEN})
+    .reply(200, {"address":"mjw2BcBvNKkgLvQyYhzRERRgWSUVG7HHTb","total_received":182464441,"total_sent":600000,"balance":181864441,"unconfirmed_balance":0,"final_balance":181864441,"n_tx":4,"unconfirmed_n_tx":0,"final_n_tx":4,"txrefs":[{"tx_hash":"b71b74674e5ca38c45bfbd0a84172a991a8e67513902f5ef31c9ca6b00d0527c","block_height":1180438,"tx_input_n":-1,"tx_output_n":1,"value":181864441,"ref_balance":363928882,"spent":false,"confirmations":19583,"confirmed":"2017-08-29T10:42:28Z","double_spend":false}],"tx_url":"https://api.blockcypher.com/v1/btc/test3/txs/"},
```
Comment (kulpreet) [inline tests/fetch_transaction_history.test.js]: Can you format the JSON in the the reply so it appears on multiple lines and is easy to read for humans.

**Review by kulpreet — COMMENTED** [review]
@Venumteam Do we need the lib folder to be inside this repo? I don't want this lib folder in here as this repo has no need for this.

File: tests/fetch_transaction_history.test.js (hunk 2 above)
Comment (Liubyvyi) [reply tests/fetch_transaction_history.test.js]: yes of course

**Pull request merged**